examples/font2svg.rs

```diff
@@ -1,10 +1,13 @@
 use std::io::Write;
 use std::path::PathBuf;
+use base64::engine::general_purpose::STANDARD;
 
 use ttf_parser as ttf;
+use ttf_parser::colr::{ClipBox, Paint};
+use ttf_parser::Transform;
 
 const FONT_SIZE: f64 = 128.0;
-const COLUMNS: u32 = 100;
+const COLUMNS: u32 = 20;
```

Comment:
Is it necessary? I don't mind, but it would look weird for normal fonts.

Reply:
No sure, I can revert it, it was just so I can inspect it better.